### PR TITLE
Chore/fix build

### LINKED ISSUE #146: You may need an appropriate loader to handle this file type: Module parse failed: Unexpected token (34:2)
Hi

After installing the modules and doing the setup I got:

```js
./node_modules/@buffetjs/styles/src/components/LoadingIndicator/index.js 34:2
Module parse failed: Unexpected token (34:2)
You may need an appropriate loader to handle this file type.
| 
| const LoadingIndicator = ({ small }) => (
>   <Loader small={small}>
|     <div />
|   </Loader>
```

The code that triggered it was simply:

```js
import {
  Enumeration,
  Table
} from '@buffetjs/core';
```

I'm not using strapi, I assume the library works standalone, right?

Can you add to the documentation the versions of the packages and the babel + webpack settings for this library to work?

I'm using [razzle's](https://github.com/jaredpalmer/razzle) default webpack and, if I'm not mistaken, they use the create-react-app boilerplate

I tried to add what you have in your babel.config.js and does not work, with or without it.

Relevant Packages:

        "react": "^16.8.6",
        "@babel/runtime": "7.4.4",
        "@buffetjs/core": "^3.2.0",
        "@buffetjs/hooks": "^3.2.0",
        "@buffetjs/styles": "^3.2.0",
        "prop-types": "^15.7.2",
        "react-dom": "^16.8.6",
        "react-is": "^16.13.1",
        "react-router-dom": "^5.0.0",
        "styled-components": "^5.1.1",

dev:
```
   "@babel/core": "7.4.5",
   "@babel/preset-env": "7.4.5",
```

Thanks!

### PR DESCRIPTION
**Pull request by soupette**
This PR changes the way we build the packages.

### TODO
- [x] Webpack setup 
- [x] Prettier setup
- [x] Stylelint setup
- [x] Eslint setup
- [x] Jest setup
- [x] Update contributing
- [ ] Dev mode

Fixes #146

### CODE REVIEW

File: packages/buffetjs-styles/src/components/InputNumber/index.js [hunk 1]
```diff
@@ -9,7 +9,10 @@ import colors from '../../assets/styles/colors';
 import sizes from '../../assets/styles/sizes';
 import img from '../../assets/icons/icon_select.svg';
 
+// TODO
```
Comment (HichamELBSI) [inline packages/buffetjs-styles/src/components/InputNumber/index.js:12]: Do you still need this comment ?

**Comment by HichamELBSI** [comment]
Sounds good to me. 
The only thing I have thought about is the parallel scripts in the package.json files. 
I usually use "npm-run-all" to run scripts in parallel but I don't really know if it will be useful here. 
Maybe it could be interesting to test all those parallel scripts in other OS.

File: packages/buffetjs-styles/src/components/InputNumber/index.js (hunk 1 above)
Comment (soupette) [reply packages/buffetjs-styles/src/components/InputNumber/index.js:12]: Yes this one is needed we need to update the file so we can pass the stylelint but it needs to be done in another PR.

**Comment by soupette** [comment]
> I usually use "npm-run-all" to run scripts in parallel but I don't really know if it will be useful here.
@alexandrebodin any thought on this?

**Comment by alexandrebodin** [comment]
From what I have seen before my holidays the build looks good overall

**Comment by soupette** [comment]
> From what I have seen before my holidays the build looks good overall

I'll make a release so we can test it on strapi then. Do you think I should merge it first?

**Comment by alexandrebodin** [comment]
You can create a develop branch and merge it but that's just for convention

**Pull request merged**